### ISSUE TITLE
Startup crash with Embers Rekindled

### ISSUE THREAD

**Issue by IdrisQe** [PersonTheCat/ore_stone_variants#41]
So, upon adding Embers to my modpack, I now cannot start the game due to an OSV crash. The issue appears to be a typo. I'm using the pre-release version you gave me to fix the Immersive Engineering Ore problem. enbers_silver_ore is, expectedly, not a thing that exists, as I have Embers... Not Enbers.
Just something that should probably be remedied before the official release.
```
---- Minecraft Crash Report ----
// On the bright side, I bought you a teddy bear!

Time: 2018-10-05 18:52:52 PDT
Description: Initializing game

java.lang.RuntimeException: Error: Unable to find properties or block with name "enbers_silver_ore"
    at personthecat.mod.properties.OreProperties.propertiesOf(OreProperties.java:131)
    at personthecat.mod.properties.PropertyGroup$Builder.build(PropertyGroup.java:352)
    at personthecat.mod.properties.PropertyGroup$Builder.buildAll(PropertyGroup.java:340)
    at personthecat.mod.init.BlockInit.init(BlockInit.java:35)
    at personthecat.mod.util.handlers.RegistryHandler.onBlockRegister(RegistryHandler.java:49)
    at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_368_RegistryHandler_onBlockRegister_Register.invoke(.dynamic)
    at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
    at net.minecraftforge.fml.common.eventhandler.EventBus$1.invoke(EventBus.java:144)
    at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:182)
    at net.minecraftforge.registries.GameData.fireRegistryEvents(GameData.java:750)
    at net.minecraftforge.fml.common.Loader.preinitializeMods(Loader.java:628)
    at net.minecraftforge.fml.client.FMLClientHandler.beginMinecraftLoading(FMLClientHandler.java:252)
    at net.minecraft.client.Minecraft.init(Minecraft.java:466)
    at net.minecraft.client.Minecraft.run(Minecraft.java:3929)
    at net.minecraft.client.main.Main.main(SourceFile:123)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196)
    at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231)
    at org.multimc.EntryPoint.listen(EntryPoint.java:143)
    at org.multimc.EntryPoint.main(EntryPoint.java:34)
```

**Comment by PersonTheCat** [comment]
Thanks. There was another post about this on [CurseForge](https://minecraft.curseforge.com/projects/ore-stone-variants/issues/45). I do make a fair amount of typos, unfortunately. This has been fixed and will be out sometime soon.

**Comment by IdrisQe** [comment]
Oh, okay, good to know! I was about to post that I found the typo in PropertyGroups.java but I guess that's not needed.
```
			new Builder("embers", true, new String[]
			{
			 	"embers_aluminum_ore", "embers_copper_ore", "embers_lead_ore",
			 	"embers_nickel_ore", "enbers_silver_ore", "embers_tin_ore"
}),
```

**Comment by PersonTheCat** [comment]
Haha yeahhh. I definitely appreciate it, though.

**Comment by PersonTheCat** [comment]
Fixed in 4.2. That'll be out in a minute. Let me know if you need anything else or have any idea for improving the mod. Thanks again for your support!
